src/Pages/Login/Login.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react'
+import './Login.scss'
+
+class Login extends Component {
+  constructor() {
+    super()
+    this.state = {
+      id: '',
+      pw: '',
+      idValidation: '',
+      pwValidation: '',
+      keepLogin: false,
+      onOffBtn: false,
+    }
+  }
+
+  handleIdChange = (event) => {
+    this.setState({
+      id: event.target.value,
+    })
+  }
+
+  handlePwChange = (event) => {
+    this.setState({
+      pw: event.target.value,
+    })
+  }
+
+  handleValidation = (event) => {
+    event.preventDefault()
+    const checkId = this.state.id
+    const checkPw = this.state.pw
+    this.setState({
+      idValidation: !checkId ? '아이디를 입력해주세요.' : '',
+    })
+    this.setState({
+      pwValidation: !checkPw ? '비밀번호를 입력해주세요.' : '',
+    })
```

Comment:
하나의 setState 안에서 작성해줄 수 있답니다 
```suggestion
    this.setState({
      idValidation: !checkId ? '아이디를 입력해주세요.' : '',
     pwValidation: !checkPw ? '비밀번호를 입력해주세요.' : '',
     })
```

Reply:
하나의 setState로 수정했습니다 :-) !